libraries/AP_Math/matrix_alg.cpp

```diff
@@ -285,7 +285,7 @@ bool inverse4x4(float m[],float invOut[])
     uint8_t i;
 
 #if CONFIG_HAL_BOARD == HAL_BOARD_SITL
-    int old = fedisableexcept(FE_OVERFLOW);
+    int old = fedisableexcept(FE_INEXACT | FE_OVERFLOW);
```

Comment:
Why is this required?  Why doesn't this fail in normal SITL?

Reply:
Yeah, I can only reproduce this inside mac, and it happens every time. Instructions used by mac os for floating point calculations are different.

Reply:
OK, could you augment the commit message, please?